CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Upgrade base image to 3.15.3-1; #56
+
+### Fixed
+- Upgrade zlib to fix CVE-2018-25032; #56
+- Upgrade ssl libraries to 1.1-1.1.1n-r0 and fix [CVE-2022-0778](https://security.alpinelinux.org/vuln/CVE-2022-0778)
```

Comment:
```suggestion
- Upgrade ssl libraries to 1.1.1n-r0 and fix [CVE-2022-0778](https://security.alpinelinux.org/vuln/CVE-2022-0778)
```
See https://pkgs.alpinelinux.org/packages?name=openssl&branch=v3.15
And https://pkgs.alpinelinux.org/packages?name=libssl1.1&branch=v3.15